server/channels/api4/outgoing_oauth_connection.go

```diff
@@ -25,6 +26,8 @@ func (api *API) InitOutgoingOAuthConnection() {
 	api.BaseRoutes.OutgoingOAuthConnection.Handle("", api.APISessionRequired(getOutgoingOAuthConnection)).Methods("GET")
 	api.BaseRoutes.OutgoingOAuthConnection.Handle("", api.APISessionRequired(updateOutgoingOAuthConnection)).Methods("PUT")
 	api.BaseRoutes.OutgoingOAuthConnection.Handle("", api.APISessionRequired(deleteOutgoingOAuthConnection)).Methods("DELETE")
+	api.BaseRoutes.OutgoingOAuthConnection.Handle("/validate", api.APISessionRequired(validateOutgoingOAuthConnectionCredentials)).Methods("POST")
+	api.BaseRoutes.OutgoingOAuthConnections.Handle("/validate", api.APISessionRequired(validateOutgoingOAuthConnectionCredentials)).Methods("POST")
```

Comment:
Do we really need two methods just to cover single vs multiple inputs?

Reply:
Both are single inputs, but one validates from the payload and the other one from the database. Users can't access the credentials once the connection is store, thus adding a new sub-endpoint to validate those as well. Both use the same logic internally (same function), the only difference is where the input comes from.

Reply:
I see, thanks for clearing that out. Would  a small comment be helpful perhaps?